src/fsverity/ioctl.rs

```diff
@@ -17,6 +17,15 @@ pub enum MeasureVerityError {
     InvalidDigestSize { expected: u16 },
 }
 
+/// Enabling fsverity failed.
+#[derive(Error, Debug, PartialEq)]
+pub enum EnableVerifyError {
```

Comment:
I wonder if this might be a bit short-sighted.  Currently this is no problem because it gets smoothed out by anyhow, but in the future if we want to start reporting errors from high-level operations on the repository, maybe we want some unified error type for that?

On the other hand, we already have MeasureVerityError and we can chain those things via `#from` in the unified error type, I suppose.

Also: we never said we won't break API, anyway!